i18n/en.json

```diff
@@ -932,6 +932,34 @@
     "translation": "shrug"
   },
   {
+    "id": "api.command_mute.desc",
+    "translation": "Mutes the notifications for this channel"
```

Comment:
Propose update to: "Turns off desktop, email and push notifications for the current channel or the [channel] specified."

Reply:
Also the slash command title should be:


`/mute [channel]`

![image](https://user-images.githubusercontent.com/14333569/32468220-9f9cd82a-c30a-11e7-87d7-e2ff2e362f07.png)